mpcs/src/basefold/commit_phase.rs

```diff
@@ -212,3 +227,80 @@ pub(crate) fn basefold_one_round_by_interpolation_weights<
         2,
     )
 }
+
+#[allow(clippy::too_many_arguments)]
+fn basefold_one_round<E: ExtensionField, Spec: BasefoldSpec<E>>(
+    pp: &<Spec::EncodingScheme as EncodingScheme<E>>::ProverParameters,
+    prover_states: &mut Vec<IOPProverState<'_, E>>,
+    challenge: Option<Challenge<E>>,
+    sumcheck_messages: &mut Vec<Vec<E>>,
+    initial_oracle: &[E],
+    transcript: &mut impl Transcript<E>,
+    trees: &mut Vec<MerkleTreeExt<E>>,
+    roots: &mut Vec<<Poseidon2ExtMerkleMmcs<E> as Mmcs<E>>::Commitment>,
```

Comment:
nitpick: it's better to name it as `commits`.

Reply:
agree this got to be better naming